overlord/snapstate/backend_test.go

```diff
@@ -603,7 +603,7 @@ apps:
 func (f *fakeSnappyBackend) ClearTrashedData(si *snap.Info) {
 	f.ops = append(f.ops, fakeOp{
 		op:    "cleanup-trash",
-		name:  si.Name(),
+		name:  si.InstanceName(),
```

Comment:
there is likely more work needed in this file, but will clear only when we add some tests to snapstate_test.go about actual instance names